test/snippet/core/detail/all_of_in_pack.cpp

```diff
@@ -1,8 +1,8 @@
 #include <type_traits>
 
-#include <seqan3/core/detail/pack_algorithm.hpp>
-#include <seqan3/core/type_list/type_list.hpp>
 #include <seqan3/core/debug_stream.hpp>
+#include <seqan3/core/detail/pack_algorithm.hpp>
+#include <seqan3/utility/type_list/type_list.hpp>
```

Comment:
Do you really need the type_list include here? I don't see a type list being used